common/src/androidTest/java/com/microsoft/identity/common/adal/internal/cache/StorageHelperTests.java

```diff
@@ -197,8 +197,9 @@ public void testKeyThumbprint() throws GeneralSecurityException, IOException {
     @Test
     public void testKeyChange() throws GeneralSecurityException, IOException {
         Context context = getInstrumentation().getTargetContext();
+        StorageHelper.LAST_KNOWN_THUMBPRINT.set("");
```

Comment:
Should this be executed before each test and move to an `@before`?

Reply:
This is really only important for this test that tries to make sure of our ability to determine that a key has changed, I would think.